brambling/templates/brambling/event/order/_attendees_ractive.html

```diff
@@ -0,0 +1,90 @@
+{# This whole template is verbatim - use ractive, not Django. #}
+{% verbatim %}
+
+<div class="row">
+	<div class="col-md-8">
+		<div class="row margin-trailer">
+			{{#each attendees }}
+				<div class='col-md-6'>
+					<div class='panel panel-primary' on-tap='assignSelectedItem(selectedItem,this)'>
+						<div class='panel-heading'>
+							<div class='panel-title'>
+								<a href='{{ id }}/'>
+									{{ full_name }}
+								</a>
+								<a class='remove-link pull-right' href='javascript://'>
+									<i class='fa fa-fw fa-trash'></i>
+								</a>
+								<a href='{{ id }}/' class="pull-right">
+									<i class='fa fa-fw fa-pencil'></i>
+								</a>
+							</div>
+						</div>
+						<ul class='list-group'>
+							{{#if this.boughtitems }}
+								{{#each this.boughtitems}}
+									<li class='list-group-item {{#if this.id == selectedItem.id}}active{{/if}}' on-tap="toggleSelectedItem(this)">{{ item_option_name }} ({{ item_name }})</li>
+								{{/each}}
+							{{else}}
+								<li class='list-group-item list-group-item-danger'>
+									Must have at least one item.
+								</li>
+							{{/if}}
+						</ul>
+					</div>
+				</div>
+				{{#if @index % 2 == 1}}
+					<div class="clearfix visible-md-block visible-lg-block"></div>
+				{{/if}}
+			{{/each}}
+			<div class='col-md-4'><a class='btn btn-primary' href='add/'>
+				<i class='fa fa-fw fa-plus'></i> Add attendee
+			</a></div>
+		</div>
+	</div>
+	<div class='col-md-4'>
+		<div class='panel panel-default' on-tap='assignSelectedItem(selectedItem,null)'>
+			<div class='panel-heading'>
+				<h4 class='panel-title'>Unassigned items</h4>
+			</div>
+			<ul class='list-group'>
+				{{#if unassigned_items }}
+					{{#each grouped(unassigned_items)}}
+						<li class='list-group-item'><strong>{{ item_name }}</strong></li>
+						{{#each this.boughtitems}}
+							<li class='list-group-item {{#if this.id == selectedItem.id}}active{{/if}}' on-tap="toggleSelectedItem(this)">
+								{{ this.item_option_name }}
+							</li>
+						{{/each}}
+					{{/each}}
+				{{else}}
+					<li class='list-group-item'>
+						No unassigned items remain.
+					</li>
+				{{/if}}
+			</ul>
+			<div class='panel-footer'>
+				{{#if selectedItem}}
+					<span class='btn btn-default btn-block disabled'>
+						Return to shop for more
+					</span>
+				{{else}}
+					<a class='btn btn-default btn-block' href='../shop'>
```

Comment:
This URL problem is going to get out of hand. We probably need to write a URL reversing API or something.